catalog.html

```diff
@@ -46,12 +48,13 @@
         и инструментов для ремонта
       </p>
       <div class="contacts-header">
-        <a class="phone-contacts-header" href="tel:+78125550555">+7 (812) 555-05-55</a>
+        <a class="phone-contacts-header" href="tel:+78125550555" aria-label="Телефон магазина Техномарт +78125550555">+7
```

Comment:
```suggestion
        <a class="phone-contacts-header" href="tel:+78125550555" aria-label="Позвонить по телефону магазина Техномарт +78125550555">+7
```

тк тут ссылка, при нажатии на которую можно позвонить

Reply:
Можно у всех телефонов поправить